source/code/plugin/in_kube_events.rb

```diff
@@ -20,6 +19,12 @@ def initialize
 
       # 30000 events account to approximately 5MB
       @EVENTS_CHUNK_SIZE = 30000
+
+      # Initializing events count for telemetry
+      @eventsCount = 0
+
+      # Initilize enable/disable normal event collection
+      @collectAllKubeEvents = false
```

Comment:
By default we are collecting all events today right? Doesn't it make sense to initialize it as true and mark false when settings is enabled?

Reply:
By default we collect only kube events with level warning. So the default value in this case would be false.